### DISCUSSION TITLE
gRPC Protocol

### DISCUSSION THREAD

**Discussion by sleipnir** [eigr/Astreu#7] [Ideas]
Here we will discuss the best semantics that the internal protocol should assume and what features there are at the protocol level we can include.

**Comment by sleipnir** [comment]
The files that describe our protocol can be found at:

1. [Core types of a messages](https://github.com/eigr/Astreu/blob/master/priv/protos/astreu/protocol/protocol.proto) 
2. [Broker management](https://github.com/eigr/Astreu/blob/master/priv/protos/astreu/manager/manager.proto)
3. [Subscriber semantics](https://github.com/eigr/Astreu/blob/master/priv/protos/astreu/consumer/subscriber.proto)
4. [Producer semantics](https://github.com/eigr/Astreu/blob/master/priv/protos/astreu/producer/publisher.proto)

ping @marcellanz

**Reply by marcellanz** [reply]
Thanks Adriano, I'll review the procotol.
